opentelemetry-api/src/opentelemetry/trace/span.py

```diff
@@ -335,11 +335,11 @@ def to_header(self) -> str:
         return ",".join(key + "=" + value for key, value in self._dict.items())
 
     @classmethod
-    def from_header(cls, header_list: typing.List[str]) -> "TraceState":
+    def from_header(cls, header: str) -> "TraceState":
```

Comment:
W3C specification https://www.w3.org/TR/trace-context/#tracestate-header-field-values

> Tracestate MAY be sent or received as multiple header fields. Multiple tracestate header fields MUST be handled as specified by RFC7230 Section 3.2.2 Field Order. The tracestate header SHOULD be sent as a single field when possible, but MAY be split into multiple header fields. When sending tracestate as multiple header fields, it MUST be split according to RFC7230. When receiving multiple tracestate header fields, they MUST be combined into a single header according to RFC7230.

I believe we should support the list of headers.